rest/config_test.go

```diff
@@ -1656,7 +1657,12 @@ func TestLoadJavaScript(t *testing.T) {
 			}()
 			js, err := loadJavaScript(inputJavaScriptOrPath, test.insecureSkipVerify)
 			if test.errExpected != nil {
-				require.True(t, errors.As(err, &test.errExpected)) // nolint: govet // CBG-2242
+				errx509Type := x509.UnknownAuthorityError{}
+				if test.errExpected == errx509Type {
+					require.ErrorContains(t, err, assertX509UnknownAuthority(test.errExpected))
+				} else {
+					require.ErrorContains(t, err, test.errExpected.Error())
+				}
 			}
 			assert.Equal(t, test.jsExpected, js)
```

Comment:
It looks like this test never made sure that we get no error when `test.ErrExpected == nil`

Maybe an extra `else` to cover this? Similar applies for all of the other tests we're touching. below

```suggestion
			} else {
				require.NoError(t, err)
				assert.Equal(t, test.jsExpected, js)
			}
```